### ISSUE TITLE
Change disk size does not work

### ISSUE THREAD

**Issue by martin-huber** [cloudsigma/terraform-provider-cloudsigma#20]
# Bug report

### Description
If I try to update a disk, I do a plan and apply. Terraform gives the following output:
```
 # cloudsigma_drive.drive_mdm_mvp_demo will be updated in-place
  ~ resource "cloudsigma_drive" "drive_mdm_mvp_demo" {
        id           = "37415871-9693-4d4e-88fe-8642033c4a52"
        name         = "mdm-mvp-demo"
      ~ size         = 53687091200 -> 85899345920
        tags         = [
            "de7528f3-4568-405d-89a2-9fd8c38cdba1",
        ]
        # (6 unchanged attributes hidden)
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

cloudsigma_drive.drive_mdm_mvp_demo: Modifying... [id=37415871-9693-4d4e-88fe-8642033c4a52]
cloudsigma_drive.drive_mdm_mvp_demo: Modifications complete after 3s [id=37415871-9693-4d4e-88fe-8642033c4a52]
```

But if I check the VM and the disk then, the size is not changed.
I tried this with the VM running, as well with the stopped VM. In both cases the disk size was not change.

If I run "plan" or "apply" again it again shows the same output as above.

### Terraform Version
Terraform v0.14.6
+ provider registry.terraform.io/cloudsigma/cloudsigma v1.3.0

### Affected Resource(s)

* cloudsigma_drive

### Terraform Configuration Files
@pavel-github  already should have our configuration

### Expected Behavior
If the VM is running, it should have been stopped, the disk size updated and the VM started again

### Actual Behavior
The VM is not restarted, the disk is not resized

!--- Please keep this note for the community --->
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
  to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by pavel-github** [comment]
👋 hey @martin-huber,
we published a new release [1.4.0](https://github.com/cloudsigma/terraform-provider-cloudsigma/releases/tag/v1.4.0) with the fix for this issue.
Could you check it if your issue is solved?

**Comment by pavel-github** [comment]
I'm going to close this issue. @martin-huber feel free to re-open it if you experience the same behaviour.
Thanks.
